developer-docs-site/docs/tutorials/first-transaction-sdk.md

```diff
@@ -0,0 +1,209 @@
+* The additional 4 coins of gas paid for by Alice to make that transfer
+
+## Step 4: The SDK in Depth
+
+The example file leverages helper functions to interact with the [REST API][rest_spec]. This section reviews each of the calls and gives insights into functionality.
+
+### Step 4.1: Initializing the Clients
+
+In the first step, the example initializes both the REST and Faucet clients. The REST client interacts with the REST API, whereas the Faucet client is a devnet service for creating and funding accounts.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```
+rest_client = RestClient(NODE_URL)
+faucet_client = FaucetClient(FAUCET_URL, rest_client)
+```
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+  </TabItem>
+</Tabs>
+
+:::tip
+
+The URLs for both services, by default, point to our devnet services, however, they can be configured via setting following environmental variables: `APTOS_NODE_URL` and `APTOS_FAUCET_URL`.
+
+:::
+
+### Step 4.2: Creating local accounts
+
+The next step is to create two accounts from the locally. [Accounts][account_basics] represent both on and off-chain state. The off-chain state consists of an address and the public, private key pair used to authenticate ownership. This step demonstrates how to generate that off-chain state.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```
+alice = Account.generate()
+bob = Account.generate()
+```
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+  </TabItem>
+</Tabs>
+
+### Step 4.3: Creating blockchain accounts
+
+In Aptos, each account must have an on-chain representation in order to support receive tokens, coins, and interacting in other DApps. In Aptos, an account represents a medium for storing assets, hence it must be explicitly created. This example leverages the Faucet to create and fund Alice's account and to just create Bob's:
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```
+faucet_client.fund_account(alice.address(), 20_000)
+faucet_client.fund_account(bob.address(), 0)
+```
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+  </TabItem>
+</Tabs>
+
+### Step 4.4: Reading balances
+
+In this step, the SDK translates a single call into the process of querying a resource and reading a field from that resource.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```
+print(f"Alice: {rest_client.account_balance(alice.address())}")
+print(f"Bob: {rest_client.account_balance(bob.address())}")
+```
+
+Behind the scenes, the SDK queries the CoinStore resource for the AptosCoin and reads the current stored value:
+```
+def account_balance(self, account_address: str) -> int:
+    """Returns the test coin balance associated with the account"""
+    return self.account_resource(
+        account_address, "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>"
+    )["data"]["coin"]["value"]
+```
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
```

Comment:
I thought we were going to do the whole page as one language?

Reply:
I'm very much against that approach. I think that will just result in low quality tutorials that quickly get out of date.